src/test/rustdoc/description.rs

```diff
@@ -0,0 +1,19 @@
+#![crate_name = "foo"]
+//! # Description test crate
+//!
+//! This is the contents of the test crate docstring. It should not show up in the description.
+
+// @matches 'foo/index.html' '//meta[@name="description"]/@content' 'Description test crate'
```

Comment:
Since you're not using regex, `@has` is the way to go.

Reply:
Okay, that's fixed.